tools/flang1/flang1exe/exterf.c

```diff
@@ -2097,18 +2097,40 @@ export_dt(int dtype)
       }
     } else /* 'null' descriptor */
       lzprintf(outlz, " %d", 0);
+
+    /* If align pragma value is smaller than its original alignment,
+     * then align pragma should have no effect
+     */
+    if (DTA(dtype) == alignment(dtype))
```

Comment:
The comment seems inconsistent with the code. Should it say "not equal to" instead of "smaller than"?